packages/button/README.md

```diff
@@ -1,44 +1,41 @@
+| Icon only       | Use when an icon is enough to represent the action, such as "Create a Service". |
+| Icon with label | Use when an icon by itself is not obvious enough, such as "Kill Task".          |
 
-### Types
+## Button States
 
-**Primary**: Primary call to action button. Should stand out above all other actions. Do not put two primary buttons next to each other. Examples include Save, Deploy.
+Buttons can have different states depending on their availability and whether an action is being processed:
 
-**Secondary**: Most buttons on a page are likely this style.
+| State      | Purpose                                                                                                                                          |
+| ---------- | ------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Disabled   | Disable a button when the action is not applicable but we want to show the user that it may be possible. Disabled buttons should have a tooltip. |
+| Processing | When a user clicks or taps a button and an API request is initiated, provide feedback so they know their action was received.                    |
+| Hidden     | Hide a button when there is no way a user can interact with the button.                                                                          |
 
-**Warning**: Warning buttons are used to advise caution.
-
-**Danger**: Destructive buttons are used to highlight something dangerous that can't be undone e.g. "Delete".
-
-**Icon only**: Use when an icon is enough to represent the action e.g. "Create a Service". Should be accompanied by a tooltip.
-
-**Icon with label**: If there is space and we feel an icon by itself is not obvious enough e.g. "Kill Task"
-
-### States
-
-**Disabled**: Disable a button when the action is not applicable but we want to show the user that it may be possible. Disabled buttons should have a tooltip.
-
-**Processing**: When user clicks or taps a button and an API request is initiated we should provide feedback so they know their action was received.
-
-**Hidden:** Hide a button when there is no way a user can interact with the button.
-
-## Using buttons as links
+## Using Buttons as Links
 
 Sometimes there is a button that is actually a link to somewhere else. If the appropriate props are set, the button is rendered as an anchor tag, but it is styled like a button.
 
 It is not recommended to use a button as a link when it triggers a dropdown. It would be frustrating if a user clicked a button and expected to see a dropdown appear, but they are taken to a new page.
 
-## Button content
+## Button Content
+
+Button labels should be concise and descriptive, tell users what will happen when they click the button.
 
-Button labels tell users what will happen when they click the button. Use verbs that describe the action. Acceptable labels include:
+Examples of acceptable labels:
 
 - New \$object
```

Comment:
What's the `\$object` here mean?

Reply:
🤷🏻‍♀️  That was there previously so I left it. I believed it to just indicate you can insert something there `Create \$object` => `Create Workspace`. I can try to think of a different way to communicate that.